From 2c7b4e70df8a3e0f05794a4f0f52df2a4b0d20ef Mon Sep 17 00:00:00 2001
From: Stanislav Kholmanskikh <stanislav.kholmanskikh@bell-sw.com>
Date: Fri, 25 Jul 2025 10:08:27 +0000
Subject: [PATCH] Set major version in SONAME to 2

The apk-tools version is 2.12.14, so it looks like a typo,
that it provided the 3.12.0 SONAME.
---
 src/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile b/src/Makefile
index 186823f..d81f9f1 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -16,7 +16,7 @@ ZLIB_CFLAGS		:= $(shell $(PKG_CONFIG) --cflags zlib)
 ZLIB_LIBS		:= $(shell $(PKG_CONFIG) --libs zlib)
 
 # Dynamic library
-libapk_soname		:= 3.12.0
+libapk_soname		:= 2.12.0
 libapk_so		:= $(obj)/libapk.so.$(libapk_soname)
 libapk.so.$(libapk_soname)-objs := \
 	common.o database.o package.o commit.o solver.o \
-- 
2.50.1

